addons/web/static/tests/legacy/views/list_view_tests.js

```diff
@@ -2362,9 +2362,10 @@ QUnit.module("Views", (hooks) => {
         ]);
     });
 
-    QUnit.test("deletion of record is disabled when groupby m2m field", async function (assert) {
+    QUnit.test("enabling archive, unarchive, and delete in list when groupby m2m field", async function (assert) {
```

Comment:
split this test into 3 (or 4 if we also enable it for duplicate), and perform the action (one per test) => assert rpcs and check that the view is in the correct state after the action (i.e. that everything as been correctly reloaded)